.travis.yml

```diff
@@ -32,10 +33,6 @@ before_script:
   - "./.travis.scripts/compile.sh"
   - mongo --eval 'tojson(db.runCommand({buildInfo:1}))'
 
-notifications:
-  email:
-  - bjori@mongodb.com
```

Comment:
thank you. I had been wondering for a long time why I get getting build error mails